build_scripts/build_usd.py

```diff
@@ -64,6 +64,9 @@ def PrintCommandOutput(output):
         sys.stdout.write(output)
 
 def PrintError(error):
+    if verbosity >= 3 and getattr(sys, "exc_value", None) is not None:
```

Comment:
Thanks for this PR, @elrond79!  exc_value has been deprecated for quite a while (https://docs.python.org/2/library/sys.html#sys.exc_value), any chance there's a formulation here with exc_info() that you'd be happy with?

Reply:
Oh, wow... how did I not know that exc_value/etc had been deprecated?? It's not like I've ever used python-1.5!

Anyway.. yeah, updated the PR to use exc_info. Thanks!

Reply:
@sunyab never misses a beat :)  Thanks so much @elrond79!